package/MDAnalysis/core/universe.py

```diff
@@ -1436,6 +1443,27 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
 
         return cls(mol, **kwargs)
 
+    def guess_TopologyAttr(self, context, to_guess):
```

Comment:
MYbe call it guess_TopologyAttributes (plural!) to make clear that they all are being guessed?